pymeasure/instruments/rigol/rigol_dg800.py

```diff
@@ -0,0 +1,167 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2024 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+from pymeasure.instruments import Instrument, Channel, SCPIMixin
+from pymeasure.instruments.validators import truncated_discrete_set
+
+
+class VoltageChannel(Channel):
+    """A channel of the signal generator"""
+
+    output_enabled = Channel.control(
+        ":OUTP{ch}?",
+        ":OUTP{ch} %s",
+        """Control the status of the channel.
+         True if the channel is on and False if not.""",
+        validator=truncated_discrete_set,
+        values={True: "ON", False: "OFF"},
+        map_values=True,
+    )
+    load = Channel.control(
```

Comment:
It is good to separate properties with an empty line.

```suggestion
    )

    load = Channel.control(
```